### ISSUE TITLE
How to use it in mmsegmentation?

### ISSUE THREAD

**Issue by endaoguansanlu** [wangchi95/active-boundary-loss#6]
Thank you so much for your excellent work! I am very interested in the loss function you proposed, but I don't know how to use it in mmsegmentation, can you please guide me? Looking forward to your reply!

**Comment by wangchi95** [comment]
Thanks for your interest in our work. For ease of use, you can add it to `mmsegmentation/mmseg/models/losses/` and modify the config file. Furthermore, you can speed it up by moving the process of distance map generation to the dataloader (in this way, maybe you need to rewrite some code of `mmcv`).
